src/views/publish/measure-preview.ejs

```diff
@@ -24,29 +24,29 @@
                     <% } %>
                 </div>
             </div>
-
-            <div class="govuk-grid-row">
-                <div class="govuk-grid-column-full">
-                    <form method="post" role="continue" enctype="multipart/form-data">
-                        <fieldset class="govuk-fieldset">
-                            <div class="govuk-form-group">
-                                <label class="govuk-label govuk-label--m" for="csv">
-                                    <%= t('publish.upload.form.file.label') %>
-                                </label>
-                                <input class="govuk-file-upload" id="csv" name="csv" type="file" placeholder="Upload Data Files!" accept=".csv,.parquet,.json,.xls,.xlsx,.gz,text/csv,application/vnd.apache.parquet,application/parquet,application/json,application/vnd.ms-excel,application/vnd.openxmlformats-officedocument.spreadsheetml.sheet,application/msexcel,application/gzip">
-                            </div>
-                            <div class="govuk-form-group">
-                                <div class="govuk-button-group">
-                                    <button type="submit" name="confirm" value="true" class="govuk-button" data-module="govuk-button" style="vertical-align: unset;" data-prevent-double-click="true">
-                                        <%= t('buttons.upload_csv') %>
-                                    </button>
-                                </div>
+        <% } %>
+
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-full">
+                <form method="post" role="continue" enctype="multipart/form-data">
+                    <fieldset class="govuk-fieldset">
+                        <div class="govuk-form-group">
+                            <label class="govuk-label govuk-label--m" for="csv">
+                                <%= t('publish.upload.form.file.label') %>
+                            </label>
+                            <input class="govuk-file-upload" id="csv" name="csv" type="file" placeholder="Upload Data Files!" accept=".csv,.parquet,.json,.xls,.xlsx,.gz,text/csv,application/vnd.apache.parquet,application/parquet,application/json,application/vnd.ms-excel,application/vnd.openxmlformats-officedocument.spreadsheetml.sheet,application/msexcel,application/gzip">
```

Comment:
Can we move this list of supported formats to a const somewhere please.

Reply:
As you request this has now been moved out to an enum.